contracts/interfaces/IIndexPool.sol

```diff
@@ -32,40 +35,54 @@ interface IIndexPool {
     uint256 tokenAmountOut
   );
 
+  /** @dev Emitted when underlying tokens are deposited for pool tokens. */
   event LOG_JOIN(
     address indexed caller,
     address indexed tokenIn,
     uint256 tokenAmountIn
   );
 
+  /** @dev Emitted when pool tokens are burned for underlying. */
   event LOG_EXIT(
     address indexed caller,
     address indexed tokenOut,
     uint256 tokenAmountOut
   );
 
+  /** @dev Emitted when a token's weight updates. */
   event LOG_DENORM_UPDATED(address indexed token, uint256 newDenorm);
 
+  /** @dev Emitted when a token's desired weight is set. */
   event LOG_DESIRED_DENORM_SET(address indexed token, uint256 desiredDenorm);
 
+  /** @dev Emitted when a token is unbound from the pool. */
   event LOG_TOKEN_REMOVED(address token);
 
+  /** @dev Emitted when a token is unbound from the pool. */
```

Comment:
Fix comment

Reply:
Responses left in <https://github.com/indexed-finance/indexed-core/pull/68>